### PR TITLE
quincy: doc/cephfs/add-remove-mds: added cephadm note, refined "Adding an MDS"

### PR DESCRIPTION
**Pull request by dparmar18**
Backport tracker: https://tracker.ceph.com/issues/55239
Backport of https://github.com/ceph/ceph/pull/45639

Description: 
-  Add a note about using cephadm for setting up the cluster and mds(s), mention
the use of ceph orchestrator if one needs to setup mds(s) manually.
- Changed the term data point to directory in point 1 under "Adding an MDS"
section for better clarity.

Fixes: https://tracker.ceph.com/issues/54551

Signed-off-by: Dhairya Parmar <dparmar@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by kotreshhr — APPROVED** [review]

**Review by kotreshhr — COMMENTED** [review]
I don't see 'cherry picked from commit <commid-id>' in the commit message. Didn't you cherry-pick this from the master commit ?

**Review by kotreshhr — COMMENTED** [review]
I don't see 'cherry picked from commit ' in the commit message. Didn't you cherry-pick this from the master commit ?

File: doc/cephfs/add-remove-mds.rst [hunk 1]
```diff
@@ -1,3 +1,10 @@
+.. note::
+   It is highly recommended to use :doc:`/cephadm/index` or another Ceph
+   orchestrator for setting up the ceph cluster. Use this approach only if you
```
Comment (anthonyeleven) [inline doc/cephfs/add-remove-mds.rst:3]: s/ceph/Ceph/ two instances

**Review by kotreshhr — COMMENTED** [review]
Also using your real name is preferred in `Signed-off` tag https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst

**Comment by dparmar18** [comment]
> I don't see 'cherry picked from commit ' in the commit message. Didn't you cherry-pick this from the master commit ?

@kotreshhr, yes I've cherry-picked it from master commit

File: doc/cephfs/add-remove-mds.rst (hunk 1 above)
Comment (dparmar18) [reply doc/cephfs/add-remove-mds.rst:3]: Making a change here would also mean I need to change it in the master commit too but I'ts already merged

**Comment by dparmar18** [comment]
> Also using your real name is preferred in `Signed-off` tag https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst

Okay, I'll make that change

File: doc/cephfs/add-remove-mds.rst (hunk 1 above)
Comment (rishabh-d-dave) [reply doc/cephfs/add-remove-mds.rst:3]: I _think_ it's okay to make the change and then add mention about the change in commit message.

**Review by rishabh-d-dave — APPROVED** [review]

**Comment by vshankar** [comment]
jenkins render docs

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
@yuriw merge after jenkins test run to completion with success.

**Pull request merged**